ingestion/src/metadata/profiler/profiler/sampler.py

```diff
@@ -12,11 +12,13 @@
 Helper module to handle data sampling
 for the profiler
 """
+# pylint: disable=consider-using-f-string,disable=inconsistent-return-statements, disable=protected-access
```

Comment:
We don't want to disable this for the full file. Whatever can be fixed in the change code, let's fix. Otherwise, just disable it in the specific line where this is raised

Reply:
sure @pmbrull